tests/faker.py

```diff
@@ -0,0 +1,23 @@
+def fake_creds():
+    return {
+        'FAKE_CHANNEL': 'C12942JF92'
+    }
+
+
+class FakeClient:
+    def __init__(self):
+        self.rtm_messages = []
+
+    def rtm_send_message(self, channel, message, attachments=None):
```

Comment:
it's a param needed for rtm_send_message function. We got it from the tests in the library itself. The slackbot has an attachment functionality. if there is no attachment given to the slackbot, then it defaults to None.